diff --git a/scripts/publish_to_pypi.sh b/scripts/publish_to_pypi.sh
index d823b0d..27d9a0c 100755
--- a/scripts/publish_to_pypi.sh
+++ b/scripts/publish_to_pypi.sh
@@ -9,4 +9,4 @@ script_dir=$(CDPATH= cd -- "$(dirname -- "$0")" && pwd)
 publisher_root=$(dirname "$script_dir")
 repo_root=$(dirname "$publisher_root")

-exec python "$script_dir/publish_to_pypi.py" "$@"
\ No newline at end of file
+exec python "$script_dir/publish_to_pypi.py" "$@"
